src/ConsumeOwnEventsService.cpp

```diff
@@ -0,0 +1,147 @@
+// Copyright (C) Martin Da Costa 2023 (martindc.merg@gmail.com)
+// This file is part of VLCB-Arduino project on https://github.com/SvenRosvall/VLCB-Arduino
+// Licensed under the Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International License.
+// The full licence can be found at: http://creativecommons.org/licenses/by-nc-sa/4.0/
+
+#include <Streaming.h>
+#include "ConsumeOwnEventsService.h"
+#include <vlcbdefs.hpp>
+
+namespace VLCB
+{
+
+void ConsumeOwnEventsService::setController(Controller *cntrl)
+{
+  this->controller = cntrl;
+  this->module_config = cntrl->getModuleConfig();
+}
+
+/// if buffer has one or more stored items
+
+bool ConsumeOwnEventsService::available(void)
+{
+  if (size > 0)
+  {
+    return true;
+  }
+  else
+  {
+  return false;
+  }
+
+ // return (size > 0);
```

Comment:
Why did you do the complex if-statement that does exactly the same as the commented out return statement below?

Reply:
I spelling things out whilst I tried to get the compiled code to actually work!  I forgot to change this back.